monitoring/statusio_maintenance.py

```diff
@@ -0,0 +1,485 @@
+        default: false
+    maintenance_notify_72_hr:
+        description:
+            - Notify subscribers 72 hours before maintenance start time
+        required: false
+        default: false
+    maintenance_notify_24_hr:
+        description:
+            - Notify subscribers 24 hours before maintenance start time
+        required: false
+        default: false
+    maintenance_notify_1_hr:
+        description:
+            - Notify subscribers 1 hour before maintenance start time
+        required: false
+        default: false
+    maintenance_id:
+        description:
+            - The maintenance id number when deleting a maintenance window
+        required: false
+        default: None
+    minutes:
+        description:
+            - The length of time in UTC that the maintenance will run \
+            (starting from playbook runtime)
+        required: false
+        default: 10
+    start_date:
+        description:
+            - Date maintenance is expected to start (Month/Day/Year) (UTC)
+            - End Date is worked out from start_date + minutes
+        required: false
+        default: None
+    start_time:
+        description:
+            - Time maintenance is expected to start (Hour:Minutes) (UTC)
+            - End Time is worked out from start_time + minutes
+        required: false
+        default: None
+'''
+
+EXAMPLES = '''
+# Create a maintenance window for 10 minutes on server1.example.com, with
+automation to stop the maintenance.
+- statusio_maintenance:
+      title: "Router Upgrade from ansible"
+      desc: "Performing a Router Upgrade"
+      components: "server1.example.com"
+      api_id: "api_id"
+      api_key: "api_key"
+      statuspage: "statuspage_id"
+      maintenance_notify_1_hr: true
+      automation: true
+
+# Create a maintenance window for 60 minutes on multiple hosts
+- name: "Create maintenance window for server1 and server2"
+  local_action:
+    module: statusio_maintenance
+    title: "Routine maintenance"
+    desc: "Some security updates"
+    components:
+      - "server1.example.com
+      - "server2.example.com"
+    minutes: "60"
+    api_id: "api_id"
+    api_key: "api_key"
+    statuspage: "statuspage_id"
+    maintenance_notify_1_hr: true
+    automation: true
+
+# Create a future maintenance window for 24 hours to all hosts inside the
+# Primary Data Center
+- statusio_maintenance:
+      title: Data center downtime
+      desc: Performing a Upgrade to our data center
+      components: "Primary Data Center"
+      api_id: "api_id"
+      api_key: "api_key"
+      statuspage: "statuspage_id"
+      start_date: "01/01/2016"
+      start_time: "12:00"
+      minutes: 1440
+
+# Delete a maintenance window
+- statusio_maintenance:
+     title: "Remove a maintenance window"
+     maintenance_id: "561f90faf74bc94a4700087b"
+     statuspage: "statuspage_id"
+     api_id: "api_id"
+     api_key: "api_key"
+     state: absent
+
+'''
+
+RETURN = '''
+changed:
```

Comment:
removed, but now fails CI. I noticed some modules have RETURN whilst others don't (in the PR queue). Not sure what to put in the RETURN bit instead.